src/lyk.h

```diff
@@ -0,0 +1,19 @@
+#include "lobject.h"
```

Comment:
This header should probably have a guard to ensure the header can be included more than once (and the second inclusion is a no-op).

The idiom usually looks like:
```
#ifndef __LYK_H
#define __LYK_H

... contents of header here ...

#endif // __LYK_H
 ```

Reply:
done :point_right: 9d00f0c7c8e12a34d6c3c7df5cfb135687790e61